Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM rust:1.50.0 AS builder
```

Comment:
It's weird that the Docker build doesn't require Rust 1.53.0, maybe because it's `portable`.

One of the Altair branches bumps this to 1.52.1 anyway, I'll make sure it goes to 1.53